web/profiles/sdd/config/install/user.role.sysadmin.yml

```diff
@@ -1,8 +1,8 @@
 langcode: en
 status: true
 dependencies: {  }
-id: administrator
-label: Administrator
+id: sysadmin
```

Comment:
It needs to check admin role in user settings

Reply:
Quite true) Was tested and looking all good atm : 
![image](https://user-images.githubusercontent.com/20738264/77152985-3cc05680-6a99-11ea-857a-2a6e9cb83954.png)

A point to keep an eye out for though